src/tcp_info.c

```diff
@@ -217,6 +217,20 @@ get_pmtu(struct iperf_interval_results *irp)
 #endif
 }
 
+/*************************************************************/
+/*
+ * Return number of reordering events seen.
+ */
+long
+get_reorder(struct iperf_interval_results *irp)
+{
+#if defined(linux) && defined(TCP_REPAIR_ON)
```

Comment:
I'm not clear on the rationale for checking for the presence of `TCP_REPAIR_ON`. Is this similar to how we check for `TCP_MD5SIG`, where it's a change that happened in the kernel about the same time as the thing you really care about (`tcpi_reord_seen`)? Just trying to understand. A comment somewhere (along the lines of what's in `has_tcpinfo_retransmit()`) might be helpful.

Reply:
Excellent, thank you!